code/modules/clothing/gloves/miscellaneous.dm

```diff
@@ -220,8 +220,8 @@
 	parry_max_attacks = INFINITY
 	parry_failed_cooldown_duration = 2.25 SECONDS
 	parry_failed_stagger_duration = 2.25 SECONDS
-	parry_cooldown = 0
-	parry_failed_clickcd_duration = 0
+	parry_cooldown = 3 SECONDS
```

Comment:
3 seconds is a little overkill of a nerf considering this is a traitor item, don't you think?